addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -221,4 +222,4 @@ def _check_unique_vendor_number(self):
                 ('state', '!=', 'cancel'),
             ]
             if rec.search(domain):
-                raise ValidationError(_('Vendor bill number must be unique per vendor and company.'))
+                raise ValidationError(_('Document %s must be unique per partner and company.', rec.name))
```

Comment:
@jco-odoo we (me and someone from the team that helps us with these topics) are already working on the po/pot file doing the translation.
As soon as we have it, I will share the file with @pepetreshere for review, and if it is ok, then it can be uploaded.